system/secret-stores/templates/cluster-store.yaml

```diff
@@ -0,0 +1,139 @@
+kind: ClusterSecretStore
+metadata:
+  name: cluster-secret-store
+spec:
+  provider:
+    onepassword:
+      connectHost: http://onepassword-connect.1password.svc.cluster.local:8080
+      vaults:
+        k8s: 1
+      auth:
+        secretRef:
+          connectTokenSecretRef:
+            namespace: 1password
+            name: onepassword-token
+            key: token
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: cloudflare-credentials
+  namespace: cloudflare-operator-system
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: cluster-secret-store
+  target:
+    creationPolicy: Owner
+  dataFrom:
+  - extract:
+      key: cloudflare--3d
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: minio-configuration
+  namespace: minio-tenant-0
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: cluster-secret-store
+  target:
+    creationPolicy: Owner
+  dataFrom:
+  - extract:
+      key: minio-configuration--xcel
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: superset-secret-key
+  namespace: superset
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: cluster-secret-store
+  target:
+    creationPolicy: Owner
+  dataFrom:
+  - extract:
+      key: superset-secret-key
+{{- end }}
+
+{{- if eq .Values.secrets_provider "aws" }}
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: cluster-secret-store
+spec:
+  provider:
+    aws:
+      service: SecretsManager
+      region: us-east-1
+      auth:
+        secretRef:
+          accessKeyIDSecretRef:
+            namespace: external-secrets
+            name: aws-credential
+            key: aws_access_key_id
+          secretAccessKeySecretRef:
+            namespace: external-secrets
+            name: aws-credential
+            key: aws_secret_access_key
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: cloudflare-credentials
+  namespace: cloudflare-operator-system
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: cluster-secret-store
+  target:
+    creationPolicy: Owner
+  dataFrom:
+  - extract:
+      key: K8S/Local/Cloudflare
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  # name: minio-console-user-credential
+  # name: minio-storage-configuration
+  name: minio-configuration
+  namespace: minio-tenant-0
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: cluster-secret-store
+  target:
+    creationPolicy: Owner
+  dataFrom:
+  - extract:
+      key: K8S/Local/MinIO
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: superset-secret-key
+  namespace: superset
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: cluster-secret-store
+  target:
+    creationPolicy: Owner
+  dataFrom:
+  - extract:
+      key: K8S/Local/Superset
```

Comment:
Having to duplicate these is not ideal however 1Password does not support `/` in item names where as it is conventional to use them in other secret stores.